GVFS/GVFS.Common/RepoMetadata.cs

```diff
@@ -91,10 +91,16 @@ public static void Shutdown()
 
         public string GetCurrentDiskLayoutVersion()

```

Comment:
Rather than calling `TryGetOnDiskLayoutVersion` you can just use the values from `GVFSPlatform` (this method was previously just formatting the hardcoded version numbers).

Reply:
Can you elaborate on that?  Shouldn't it do check similar to what TryGetOnDiskLayoutVersion is doing?  Are there cases where a value could be missing and an error is more preferable?

Reply:
> Can you elaborate on that?

I wasn't sure what the motivation was for changing the behavior of `GetCurrentDiskLayoutVersion` as part of the changes in this PR.

`GetCurrentDiskLayoutVersion` is simply a helper method for formatting `CurrentMajorVersion` and `CurrentMinorVersion`, it doesn't check what's on disk.

(In fact, it's only called in one spot and could potentially be replaced with `$"{GVFSPlatform.Instance.DiskLayout.CurrentMajorVersion}.{GVFSPlatform.Instance.DiskLayout.CurrentMinorVersion}"`).

> Are there cases where a value could be missing and an error is more preferable?

The error conditions are already being detected by the mount verb and mount process using `TryGetOnDiskLayoutVersion`.

Perhaps there's a better name for this method (or we should simply remove it) to make it clearer what it's being used for?